### ISSUE TITLE
How to cut off gradient in tensorflow2.0

### ISSUE THREAD

**Issue by jianrui1995** [tensorflow/tensorflow#41646]
I want to cut off gradient when training. For example: 
![image](https://user-images.githubusercontent.com/20520524/88247306-db6ef500-cccf-11ea-9da1-d53e5e27ebc6.png)
In picture, there are six ops. I want to loss7 only influence op5 and op6, so I need to cut off grandient flow between op2 and op5. 
Do you konw how to do it. I am looking forward your answer. thanks

**Comment by wookayin** [comment]
Is `tf.stop_gradient(op5)` what you're looking for?

**Comment by jianrui1995** [comment]
yes，**tf.stop_gradient(op5)** is what I need , Thank you very much.
